mtp/mtp_db.cpp

```diff
@@ -1,57 +1,79 @@
 // Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
 // For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
+#include <algorithm>
 #include "mtp_db.hpp"
 
 namespace mtp
 {
+    namespace filename_to_handle
+    {
+        inline Handle &getHandle(std::map<std::filesystem::path, Handle>::iterator iter)
+        {
+            return iter->second;
+        }
+    } // namespace filename_to_handle
+
+    namespace handle_to_filename
+    {
+        inline auto &getIter(std::map<Handle, std::map<std::filesystem::path, Handle>::iterator>::iterator iter)
```

Comment:
This `std::map<Handle, std::map<std::filesystem::path, Handle>::iterator>::iterator` is used in a few places and is quite long - please define an alias with `using`

Reply:
Done